README-adjustcarryforward.md

```diff
@@ -31,25 +31,32 @@ file:
 Note that the column names should be as described for `cleaned_data` the Example 
 under Quickstart.
 
-The sweep script is executed from the command line on the cleaned data file:
+The sweep script is executed from the command line on the cleaned data file for 
+Windows as follows:
 
 ```bash
 % Rscript exec/testadjustcf.R cleaned.csv
 ```
 
+For Macs/Linux, the script is executed with the following command:
+
+```bash
+% Rscript testadjustcf.R cleaned.csv
```

Comment:
When I run this script on my Mac, I use the full relative path, including `exec/`. I'm trying to recall, was there some adjustment David made to his version other than changing directories? 

Either way, do we need to say something about the current working directory?

Reply:
Hmmm, now I'm wondering that as well. He might have just navigated to that directory then? I will update this and say something about the working directory, I agree that it's important.